hello-world/apps/hello-world/pom.xml

```diff
@@ -11,7 +11,7 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 		<project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
 		<maven-compiler-plugin.version>3.8.1</maven-compiler-plugin.version>
-		<fuse.version>7.9.0.fuse-sb2-790065-redhat-00001</fuse.version>
+		<fuse.version>7.10.0.fuse-sb2-7_10_0-00014-redhat-00001</fuse.version>
 		<maven.compiler.source>1.8</maven.compiler.source>
 		<maven.compiler.target>1.8</maven.compiler.target>
```

Comment:
Jaime opened an issue today that we are targeting J1.8 while in theory all of our code is for J11 (issue #190 of the backlog). Perhaps we can change that in this PR so we don't need to do the effort of reviewing that again. Thoughts?

Reply:
Yes, I agree on this.